### DISCUSSION TITLE
Wrong API key error despite using the correct token from the dashboard

### DISCUSSION THREAD

**Discussion by hexclann** [simple-login/app#1570] [Self-hosting question]
Hello, I have created an API key from the dashboard, but when I make a request I get:
`{"error":"Wrong api key"}` error despite copy pasting the exact key from the UI.

This is the command:
`curl -s -X GET "https://app.mydomain.com/api/v5/alias/options" -H "Authorization: Bearer gcthyhtdhffqmthrrroRedactedToken" -H "Content-Type:application/json"`

**Comment by nguyenkims** [comment]
@hexclann hi the header is rather ` -H "Authentication: gcthyhtdhffqmthrrroRedactedToken"`
